lib/rucio/core/rule.py

```diff
@@ -1717,7 +1713,7 @@ def re_evaluate_did(scope, name, rule_evaluation_action, *, session: "Session"):
 
 
 @read_session
-def get_updated_dids(total_workers, worker_number, limit=100, blocked_dids=[], *, session: "Session"):
+def get_updated_dids(total_workers: int, worker_number: int, limit: int = 100, blocked_dids: Iterable[tuple[str, str]] = [], *, session: "Session") -> list[tuple[str, str]]:
```

Comment:
The query has four columns, so the return type should be `list[tuple[str, InternalScope, str, DIDReEvaluation]]`.